databases/week3/Week3Homework.sql

```diff
@@ -0,0 +1,205 @@
+SET NAMES utf8mb4;
+
+CREATE DATABASE meals_sharing;
+use meals_sharing;
+
+
+-- create table meals
+
+CREATE TABLE `meal`(
+ `id` int(10) unsigned NOT NULL auto_increment PRIMARY KEY,
+ `title` varchar(255) NOT NULL,
+ `description`text NULL default NULL,
+ `location` varchar(255) NOT NULL,
+ `When` datetime NOT NULL,
+ `max_reservations` int(10) unsigned NOT NULL,
+ `price` decimal(15,2),
+ `created_date` DATE NOT NULL
+ );
```

Comment:
Been repeated down , why?

Reply:
Had some warnings. So dropped the table and created again